cli/src/cmd/error.rs

```diff
@@ -30,12 +30,34 @@ pub enum CommandErrorKind {
     Other(String),
 }
 
+impl std::fmt::Display for CommandErrorKind {
```

Comment:
Could be replace by usage of: https://jeltef.github.io/derive_more/derive_more/display.html